universal-login-react/src/ui/TopUp/TopUp.tsx

```diff
@@ -14,7 +15,11 @@ interface TopUpProps {
 export const TopUp = ({contractAddress, startModal, onRampConfig}: TopUpProps) => {
   const [modal, setModal] = useState<TopUpComponentType>(startModal || TopUpComponentType.choose);
   if (modal === TopUpComponentType.choose) {
-    return (<TopUpChoose onMethodChoose={setModal}/>);
+    return (
+      <ModalWrapper isVisible modalPosition="bottom">
```

Comment:
Position should be got from props and have default value 'bottom'?

Reply:
Can we move it to custom css? #NextPr